### ISSUE TITLE
Do you still manage this repo?

### ISSUE THREAD

**Issue by Sarosk** [AlJohri/socialscraper#10]
Hi,
found your repo and interested with it, specially with the facebook library.
does it still work?

**Comment by AlJohri** [comment]
No, unfortunately not. If you get it working again I'd be happy to merge your changes.
